### PR TITLE
chore: Succession

### PR DESCRIPTION
**Pull request by PeerRich**
it is what it is 👁️👄👁️

### CODE REVIEW

**Review by ouwargui — APPROVED** [review]
looks accurate to me!

**Review by 0xDebabrata — APPROVED** [review]
W

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
   <h3 align="center">Cal.com (formerly Calendso)</h3>
 
   <p align="center">
-    The open-source Calendly alternative.
+    The open-source Calendly successor.
```
Comment (itsmingjie) [inline README.md:10]: hmm. maybe, i don't know

```suggestion
    The open-source Calendly killer.
```

**Review by mcnaveen — APPROVED** [review]

**Review by exception — APPROVED** [review]
LGTM!

**Review by tonilopezmr — APPROVED** [review]
LGTM 🤘🎸

**Review by ephraimduncan — APPROVED** [review]
🚀

File: README.md (hunk 1 above)
Comment (zlwaterfield) [inline README.md:10]:
```suggestion
    The Calendly dethroner.
```

**Pull request merged**